Sources/UtiliKit/Obfuscation/ObfuscatedKey.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  ObfuscatedKey.swift
+//  UtiliKit-iOS
+//
+//  Created by Russell Mirabelli on 7/23/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+// Inspired by https://github.com/onmyway133/Arcane/blob/master/Sources/Arcane/Obfuscator.swift
+
+import Foundation
+
+/// One should never commit keys directly into one's source; this is an unsafe practice.
+/// When it is impossible to avoid doing so, the key should at least be obfuscated.
+/// By using the ObfuscatedKey struct, you can build a human-readable key that nonetheless
+/// will not appear simply by running "strings" against your compiled code, and will even
+/// not appear as a string within your source code.
+struct ObfuscatedKey {
+    private let _value: String
```

Comment:
Can we lose the internal variable and use `public private(set) value: String` ?

Reply:
That's not available on a "let"; I believe that the immutability of a "let" is important to make the general builder technique make sense.

Reply:
Fair point - withdrawn.

Reply:
I can't help but feel like I'm missing something here - what does the `private let _value: String` and `public var value { return _value }` get us here over a single immutable `public let value`. As far as I can tell, we aren't mutating it or validating internally and the same-file extensions should have the same access regardless.

Reply:
Fair point. Since my merge is _still_ blocked by the license (ugh), I'll think about this and see if there is a good reason I can find.

Reply:
Reviewed, changed.